blockball-core/src/test/java/unittest/VirtualArenaServiceTest.kt

```diff
@@ -78,13 +72,28 @@ class VirtualArenaServiceTest {
         }
 
         with(goal.upperCorner) {
-            x = 150.0
-            y = 140.0
-            z = 150.0
+            x = 110.0
+            y = 130.0
+            z = 110.0
         }
 
-        Mockito.`when`(player.world).thenReturn(world)
         val classUnderTest = createWithDependencies(mockedParticleService)
+        val locations = arrayListOf<Position>()
+        for(calls in 1..2){
+            var j = goal.lowerCorner.y
```

Comment:
Duplicating the logic being tested for a test is bad practise. Still, I appreciate it that you have tried. I'll accept it anyway if you cannot come up with a better test.